### PR TITLE
fix(rn): disable pip while authorising dropbox

### PR DESCRIPTION
**Pull request by tmoldovan8x8**

### CODE REVIEW

File: react/features/dropbox/functions.native.js [hunk 1]
```diff
@@ -4,14 +4,18 @@ import { NativeModules } from 'react-native';
...
+
 /**
  * Action to authorize the Jitsi Recording app in dropbox.
  *
  * @returns {Promise<Object>} - The promise will be resolved with the dropbox
  * access token or rejected with an error.
  */
 export function _authorizeDropbox(): Promise<Object> {
-    return Dropbox.authorize();
+    setPictureInPictureDisabled(true);
```
Comment (saghul) [inline react/features/dropbox/functions.native.js:16]: this returns a promise so we should probably do a chain here

File: react/features/dropbox/actions.js [hunk 2]
```diff
@@ -1,8 +1,11 @@
 // @flow
+import Logger from 'jitsi-meet-logger';
 
 import { UPDATE_DROPBOX_TOKEN } from './actionTypes';
 import { _authorizeDropbox } from './functions';
 
+const logger = Logger.getLogger(__filename);
```
Comment (saghul) [inline react/features/dropbox/actions.js:7]: we don't use __filename, please use a logger for the feature

**Review by saghul — COMMENTED** [review]
left some comments!

File: react/features/dropbox/functions.native.js [hunk 3]
```diff
@@ -4,14 +4,22 @@ import { NativeModules } from 'react-native';
...
 /**
  * Action to authorize the Jitsi Recording app in dropbox.
  *
  * @returns {Promise<Object>} - The promise will be resolved with the dropbox
  * access token or rejected with an error.
  */
-export function _authorizeDropbox(): Promise<Object> {
-    return Dropbox.authorize();
+export async function _authorizeDropbox(): Promise<Object> {
+    setPictureInPictureDisabled(true);
```
Comment (saghul) [inline react/features/dropbox/functions.native.js:16]: you should await this

Comment (tmoldovan8x8) [reply react/features/dropbox/functions.native.js:16]: It is not async

Comment (saghul) [reply react/features/dropbox/functions.native.js:16]: It is, all native functions are

**Review by saghul — APPROVED** [review]

**Pull request merged**